Sources/SourceKitLSP/Swift/SwiftTestingScanner.swift

```diff
@@ -292,7 +292,7 @@ final class SyntacticSwiftTestingTestScanner: SyntaxVisitor {
     }
 
     let name =
-      node.name.text + "(" + node.signature.parameterClause.parameters.map { "\($0.firstName):" }.joined() + ")"
+      node.name.text + "(" + node.signature.parameterClause.parameters.map { "\($0.firstName.trimmed):" }.joined() + ")"
```

Comment:
Accessing `.text` is slightly faster than accessing `.trimmed` because it doesn’t build an intermediate node.